app/models/ability.rb

```diff
@@ -0,0 +1,19 @@
+class Ability
+  include CanCan::Ability
```

Comment:
giải thích cho anh các quyền trong này

Reply:
Dạ các quyền trong này bao gồm có guest, user và admin
- Với guest (người dùng chưa đăng nhập) thì sẽ xem được thông tin về các món ăn
-  Với user đã đăng nhập thì sẽ xem được thông tin món ăn,  xem và chỉnh sửa thông tin về chính mình, ngoài ra còn có xem, tạo mới và chỉnh sửa status của đơn hàng
- Với admin thì sẽ có quyền xem với tất cả, toàn quyền CRUD với món ăn và quyền quản lý trạng thái với đơn hàng ạ